### PR TITLE
add today's event

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _events/2020-09-19-devs-roundtable.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+title: "Galaxy Developer Roundtable"
+starts: 2020-09-19
+ends: 2020-09-19
+organiser:
+location: online
+supporters:
+---
+
+Like every other Thursday at **6 pm CET**, the Galaxy Developer Roundtable will take place. Today, [**John Chilton**](https://galaxyproject.org/people/john-chilton/) is the presenter.
```
Comment (bgruening) [inline _events/2020-09-19-devs-roundtable.md:12]: Is that not every second Thursday...?

Comment (beatrizserrano) [reply _events/2020-09-19-devs-roundtable.md:12]: means the same, left? @simonbray we need a native English speaker here ;)

Comment (bgruening) [reply _events/2020-09-19-devs-roundtable.md:12]: Let me rephrase, is "Like every second Thursday" not better as we do not know when people will read it?

Comment (beatrizserrano) [reply _events/2020-09-19-devs-roundtable.md]: changed! :)

Comment (simonbray) [reply _events/2020-09-19-devs-roundtable.md]: Sorry, I missed this. Every second, every other, either is fine.
